README.rst

```diff
@@ -0,0 +1,151 @@
+Nameko eventlog dispatcher
+==========================
+
+.. pull-quote::
+
+    `Nameko <http://nameko.readthedocs.org>`_ dependency provider that
+    dispatches log data using ``Events`` (Pub-Sub).
+
+
+.. image:: https://travis-ci.org/sohonetlabs/nameko-eventlog-dispatcher.svg?branch=master
+
+
+Usage
+-----
+
+Dispatching event log data
+``````````````````````````
+
+Include the ``EventLogDispatcher`` dependency in your service class:
+
+.. code-block:: python
+
+    from nameko.rpc import rpc
+    from nameko_eventlog_dispatcher import EventLogDispatcher
+
+
+    class FooService:
```

Comment:
My intention was to have example class names without any meaning; names that don't suggest anything about the service. I'd like to keep the existing names, if you are fine with it, so that the name of the class does not imply anything about the service purpose, and keep the focus on the dependency code that can be use in any kind of service.